packages/core/src/middleware/koa-user-log.ts

```diff
@@ -19,7 +19,7 @@ export interface LogContext {
   createdAt: number;
 }
 
-const insertLog = async (ctx: WithUserLogContext<Context>, result: UserLogResult) => {
+const recordUserLog = async (ctx: WithUserLogContext<Context>, result: UserLogResult) => {
```

Comment:
why change this name?

Reply:
it conflicts with `insertLog` in `queries/log`

Reply:
~~conflict with `insertLog` in `queries/log`~~
my bad. no need to rename it now.